lib/navigation/service/navigation_api.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:acs_upb_mobile/authentication/service/auth_provider.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:provider/provider.dart';
+
+class NavigationApi {
```

Comment:
Why is this necessary? Seems easier to just call `Provider.of<AuthProvider>(context).isAuthenticated` rather than have an entire class for this, and add a level of abstraction...

Reply:
it was my first approach and I forgot about this file. Deleted it in this commit https://github.com/student-hub/acs-upb-mobile/pull/265/commits/c0e445abfb28015aac913a15d8170ec4b81ff32d